### ISSUE TITLE
Show line number in file where error is

### ISSUE THREAD

**Issue by limsim** [jessedc/ajv-cli#34]
Hi. I ran a validation and got quite a few errors saying "data[0] should NOT have additional properties". However with no line numbers it's quite hard to work out what field is actually extra.

I think it would be a nice feature to also say what line in the file the error is referring to.

PS: Nice tool.

**Comment by epoberezkin** [comment]
additional property is present in error object in `params` property. Mapping to a line number in the actual JSON file can be done using https://github.com/epoberezkin/json-source-map - it would be an additional error reporting format (--error option).

**Comment by ffes** [comment]
Indeed, with the message `Unexpected token } in JSON at position 25272` it was not that easy to find out where the error was.
